<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createitempage</class>
 <widget class="QWidget" name="createitempage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #384b06;
font: 12pt &quot;DejaVu Serif&quot;;</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>761</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #face23;
border-radius: 12px;

</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Serif</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px;</string>
     </property>
     <property name="text">
      <string>Select Item Type:</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>60</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #9ebe41;</string>
     </property>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="createItemButton">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>20</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #384b06;
font-size: 18px;
color: white;
</string>
     </property>
     <property name="text">
      <string>Create Item</string>
     </property>
    </widget>
    <widget class="QWidget" name="commonFieldsContainer" native="true">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>120</y>
       <width>261</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>48</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Serif</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 18px;</string>
      </property>
      <property name="text">
       <string>Title:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="titleInput">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 18px;
background-color: #9ebe41;
border-radius: 20px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Serif</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 18px;</string>
      </property>
      <property name="text">
       <string>Author:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="authorInput">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 18px;
background-color: #9ebe41;
border-radius: 20px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>47</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Serif</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 18px;</string>
      </property>
      <property name="text">
       <string>Year:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>54</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Serif</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 18px;</string>
      </property>
      <property name="text">
       <string>ISBN:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="isbnInput">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>210</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 18px;
background-color: #9ebe41;
border-radius: 20px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>72</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Serif</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 18px;</string>
      </property>
      <property name="text">
       <string>Format:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>95</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Serif</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 18px;</string>
      </property>
      <property name="text">
       <string>Condition:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="yearInput">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Serif</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 18px;
background-color: #9ebe41;
border-radius: 20px;</string>
      </property>
      <property name="minimum">
       <number>1000</number>
      </property>
      <property name="maximum">
       <number>2100</number>
      </property>
      <property name="value">
       <number>2025</number>
      </property>
     </widget>
     <widget class="QComboBox" name="formatInput">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 18px;
background-color: #9ebe41;
border-radius: 20px;</string>
      </property>
      <item>
       <property name="text">
        <string>Select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hardcover</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Paperback</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Magazine</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Blu-ray</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DVD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PlayStation</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Xbox</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nintendo Switch</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="conditionInput">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 18px;
background-color: #9ebe41;
border-radius: 20px;</string>
      </property>
      <item>
       <property name="text">
        <string>Select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>New</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Excellent</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Good</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fair</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Poor</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="dynamicContainer1" native="true">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>370</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px;
background-color: #9ebe41;
border-radius: 0px;
</string>
     </property>
    </widget>
    <widget class="QWidget" name="dynamicContainer2" native="true">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>450</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px;
background-color: #9ebe41;
border-radius: 0px;
</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #9ebe41;</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Serif</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20px;

background-color: none;</string>
      </property>
      <property name="text">
       <string>Create New Item</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>321</width>
       <height>421</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-colour: none;
border: 2px dashed #000000;</string>
     </property>
    </widget>
    <zorder>widget_4</zorder>
    <zorder>widget_3</zorder>
    <zorder>widget_2</zorder>
    <zorder>label_2</zorder>
    <zorder>createItemButton</zorder>
    <zorder>commonFieldsContainer</zorder>
    <zorder>dynamicContainer1</zorder>
    <zorder>dynamicContainer2</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
